### ISSUE TITLE
Narrator fails to announce the detailslist row position details properly in firefox

### ISSUE THREAD

**Issue by SumiranAgg** [OfficeDev/office-ui-fabric-react#3834]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
Normal

Products/sites affected: (if applicable)
VSTS

#### Describe the issue:

NVDA fails to announce the detailslist row position details properly in Firefox browser

#### Actual behavior:

First row narration is "Row 0 of" <total count>. i.e. indexing starts from 0.
Also total count narrated is (actual count + 1).

#### Expected behavior:

NVDA should announce appropriate Position of the rows available on DetailsList component.

**Comment by mdmdakbar** [comment]
I am using Edge+Narrator and same issue exist

**Comment by ThomasMichon** [comment]
@SumiranAgg This issue should have been fixed by #5066. Could you re-test?

**Comment by oengusmacinog** [comment]
After some thorough review, I found that the cell position is reading out correctly with Narrator using Edge.  The row does not, but according to our design team this is the desired interaction: we don't want to get too specific at every level, just when you get down to the cell level.

As for Firefox, or any other browser other than Edge, the priority support for Narrator is Edge.  As Edge and Narrator are Microsoft products, full control over the experience is possible, unlike Firefox where there are limitations based on how they build their browser.  We do still do our best to accommodate other browsers, but for the best experience with Narrator use Edge.

Closing this issue for now.  If you have other specific issues please feel free to open a new issue.
